packages/core/src/kernelCoreTemplate.js

```diff
@@ -117,7 +116,9 @@
         const flags = parts[parts.length - 1]
         scuttleGlobalThisExceptions[i] = new RegExp(pattern, flags)
       }
-      snow(realm => performScuttleGlobalThis(realm, scuttleGlobalThisExceptions), globalRef)
+      scuttler(globalRef, {
+        scuttle: realm => performScuttleGlobalThis(realm, scuttleGlobalThisExceptions),
```

Comment:
`(cb, realm)` was proposed originally because the only "option" was to use snow, which accepts `(cb, realm)` because `cb` is required while `realm` is optional.
Now that we go with a more generic option that just snow, it makes sense to go by standard which is first `realm` and `cb` after.

I do agree with `(realm, cb, {})` however, that's a fair suggestion.

Reply:
(You can see what I mean btw by visiting `packages/browserify/examples/01-simple-js/index.html:5` on this PR)

Reply:
aea3bbc57